### PR TITLE
Import tests for custom element reactions from WebKit.

### PR DESCRIPTION
**Pull request by rniwa**

### CODE REVIEW

**Comment by TakayoshiKochi** [comment]
Is this different from #3835?

**Review by dominiccooney — COMMENTED** [review]
This generally looks good. I'm unsure how this relates to that earlier patch, does it need merging with that?

File: custom-elements/attribute-changed-callback.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+MyCustomElement.observedAttributes = ['title', 'id', 'r'];
+customElements.define('my-custom-element', MyCustomElement);
+
+test(function () {
+    var instance = document.createElement('my-custom-element');
+    argumentList = [];
+
+    instance.setAttribute('title', 'foo');
+    assert_equals(instance.getAttribute('title'), 'foo');
+    assert_equals(argumentList.length, 1);
```
Comment (rniwa) [inline custom-elements/attribute-changed-callback.html]: Sure, done that.

**Comment by rniwa** [comment]
Oh actually, https://github.com/w3c/web-platform-tests/pull/3835 already imported these tests so this is really just a refactoring now.

**Comment by TakayoshiKochi** [comment]
LGTM

**Pull request merged**